etc/parameters/default.json

```diff
@@ -477,12 +477,12 @@
     "min_step_duration": { "nanos": 100000000, "secs": 0 },
     "sole_pressure_threshold": 0.4,
     "starting_step": {
-      "foot_lift_apex": 0.009,
-      "step_duration": { "nanos": 200000000, "secs": 0 }
+      "foot_lift_apex": 0.0065,
+      "step_duration": { "nanos": 250000000, "secs": 0 }
     },
     "step_midpoint": {
-      "forward": 0.45,
-      "left": 0.35,
+      "forward": 0.5,
+      "left": 0.5,
```

Comment:
The idea behind this is to do the interpolation similar as you for side-steps. You always reach the mid-point after a fixed time and increase the other half of the interpolation (as far as I understood you)

Reply:
Yes, you understood correctly.
If I see it correctly, side walks are (with my suggested parameters) increased in the duration by 16.5 ms. Less than I wanted. I planned to increase it to from 250 ms to about 285 ms => `step_duration_increase` needs to be 1.4.

(35 ms increase -> 35 / 250 = 0.14
Duration increase is based on side step size times increase factor -> 0.1 * 1.4 = 0.14)

With that a midpoint a 125 ms would mean a value of 125 / 285 = 0.43.
(*Keeping in mind, that a separate interpolation time for the translation/rotation and the height would be better)